purchases/src/main/kotlin/com/revenuecat/purchases/models/Transaction.kt

```diff
@@ -24,9 +29,24 @@ data class Transaction(
     val purchaseDate: Date,
     val storeTransactionId: String?,
     val store: Store,
+    val displayName: String?,
+    val isSandbox: Boolean = false,
+    val originalPurchaseDate: Date?,
+    val price: Price?,
```

Comment:
Same, some of these could be breaking changes, so we might need to add a constructor without these fields to keep backwards compatibility.